### ISSUE TITLE
Following Symbolic Links

### ISSUE THREAD

**Issue by ColinHasted** [servocoder/RichFilemanager#241]
I don't know if this is an issue with file manager of my lack of understanding. I am running IIS on Windows 10 and I am using Symbolic Links so that File Manager can browse files from multiple places. If fileroot is set to the userfolder I can see the Symbolic Link folders from the web but I can't browse them (FileManager says the directory does not exist). However, if I set FileManager root to C:\ and browse down to the user folder then the symbolic links work perfectly.

**Comment by servocoder** [comment]
What connector do you use?

**Comment by ColinHasted** [comment]
PHP

**Comment by servocoder** [comment]
I use PHP connector at the RFM demo site and it refer to the symlink folder. As you can see it work perfect. Check permissions of the symlinks you create.

**Comment by ColinHasted** [comment]
Permissions seem ok as I can browse the symbolic link If I set the user folder to c:/ and then browse down from there to the symbolic link (which is in the default user folder). If I set the user folder to the default user folder without changing the symbolic link I can see it but get the error in the first post. I'll keep hunting to see if I can find the error. Else I might change to Apache as it might be IIS causing problems.
